src/dashboards/Map/MapView/index.tsx

```diff
@@ -24,63 +28,75 @@ const MapView = ({
+                        longitude: settings?.coordinates?.longitude,
+                        width: 'auto',
+                        height: window.innerHeight - 124,
+                        zoom,
+                        maxZoom,
+                        minZoom,
+                    })
+                }}
             >
-                <PositionPin size="24px" />
-            </Marker>
-            {bikeRentalStations?.map((station) => (
-                <Marker
-                    key={station.id}
-                    latitude={station.latitude}
-                    longitude={station.longitude}
-                    marker-size="large"
-                >
-                    <BicycleCapacity
-                        capacity={station.bikesAvailable ?? 0}
-                    ></BicycleCapacity>
-                </Marker>
-            ))}
-            {stopPlacesWithDepartures?.map((stopPlace) => (
+                {stopPlacesWithDepartures?.map((stopPlace) =>
+                    stopPlace.departures.length > 0 ? (
+                        <Marker
+                            key={stopPlace.id}
+                            latitude={stopPlace.latitude ?? 0}
+                            longitude={stopPlace.longitude ?? 0}
+                            offsetLeft={-50}
+                            offsetTop={-10}
+                        >
+                            <StopPlaceTag
+                                stopPlace={stopPlace}
+                                travelTimes={travelTimes}
+                            />
+                        </Marker>
+                    ) : (
+                        []
+                    ),
+                )}
+                {bikeRentalStations?.map((station) => (
+                    <Marker
+                        key={station.id}
+                        latitude={station.latitude}
+                        longitude={station.longitude}
+                        marker-size="large"
+                    >
+                        <BicycleTag
+                            bikes={station.bikesAvailable ?? 0}
+                            spaces={station.spacesAvailable ?? 0}
+                        ></BicycleTag>
+                    </Marker>
+                ))}
                 <Marker
-                    key={stopPlace.id}
-                    latitude={stopPlace.latitude ?? 0}
-                    longitude={stopPlace.longitude ?? 0}
-                    offsetLeft={-0}
-                    offsetTop={-10}
+                    latitude={viewport.latitude ?? 0}
+                    longitude={viewport.longitude ?? 0}
                 >
-                    <Tooltip
-                        placement={data > 1 ? 'top' : 'right'}
-                        stopPlace={stopPlace}
-                    >
-                        <div className="tooltip-content"></div>
-                    </Tooltip>
+                    <PositionPin size="24px" />
```

Comment:
Fikk høre på en tidligere PR at er vanlig å kun bruke tall for pixel verdier 😊 
``` size={24}``` mener jeg er syntaksen jeg ble anbefalt, dog jeg mener det skal funke med å bare fjerne `px` også!

Reply:
Fikser 😁